ckanext/canada/templates/internal/user/login.html

```diff
@@ -52,7 +52,7 @@
     {% endif %}
 
     {% block form %}
-      {{ super() }}
+      {% snippet 'user/login_form.html', action=c.login_handler, error_summary=error_summary %}
```

Comment:
you moved the snippet from user/snippets/login_form.html to user/login_form.html. Was there a reason for that?